libs/langchain-cloudflare/src/chat_models.ts

```diff
@@ -135,6 +224,20 @@ export class ChatCloudflareWorkersAI
     }
   }
 
+  override bindTools(
```

Comment:
Did we ever get around to that generic with structured output method?

Reply:
Not merged on PY side yet. Pending review